examples/vuejs/with-thirdpartyemailpassword/src/views/AuthView.vue

```diff
@@ -0,0 +1,593 @@
+<script lang="ts">
+import ThirdPartyEmailPassword from "supertokens-web-js/recipe/thirdpartyemailpassword";
+
+const websitePort = import.meta.env.VUE_APP_WEB_PORT || 3000;
+const websiteDomain = import.meta.env.VUE_APP_WEB_URL || `http://localhost:${websitePort}`;
+
+export default {
+    data() {
+        return {
+            isSignIn: true,
+            email: "",
+            password: "",
+            error: false,
+            errorMessage: "Something went wrong",
+        };
+    },
+
+    mounted() {
+        const params = new URLSearchParams(window.location.search);
+
+        if (params.has("error")) {
+            this.errorMessage = "Something went wrong";
+            this.error = true;
+        }
+    },
+
+    methods: {
+        goToSignUp() {
+            this.isSignIn = false;
+        },
+        goToSignIn() {
+            this.isSignIn = true;
+        },
+        signIn: async function (_: SubmitEvent) {
+            if (
+                !(
+                    await ThirdPartyEmailPassword.doesEmailExist({
+                        email: this.email,
+                    })
+                ).doesExist
+            ) {
+                this.errorMessage = "Email does not exist, sign up instead";
+                this.error = true;
+                return;
+            }
+
+            const response = await ThirdPartyEmailPassword.emailPasswordSignIn({
+                formFields: [
+                    {
+                        id: "email",
+                        value: this.email,
+                    },
+                    {
+                        id: "password",
+                        value: this.password,
+                    },
+                ],
+            });
+
+            if (response.status === "WRONG_CREDENTIALS_ERROR") {
+                this.errorMessage = "Invalid credentials";
+                this.error = true;
+                return;
+            }
+
+            if (response.status !== "OK") {
+                this.errorMessage = "Something went wrong";
+                this.error = true;
+                return;
+            }
+
+            window.location.assign("/");
+        },
+        validateEmail(email: string) {
+            return email
+                .toLowerCase()
+                .match(
+                    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+                );
+        },
+        signUp: async function (_: SubmitEvent) {
+            if (
+                (
+                    await ThirdPartyEmailPassword.doesEmailExist({
+                        email: this.email,
+                    })
+                ).doesExist
+            ) {
+                this.errorMessage = "Email already exists, sign in instead";
+                this.error = true;
+                return;
+            }
```

Comment:
Do not need this, since the sign up check below will return a general error implying this anyway